tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -347,13 +362,19 @@ def test_dhcp_relay_default(ptfhost, dut_dhcp_relay_data, validate_dut_routes_ex
             if not skip_dhcpmon:
                 time.sleep(18)      # dhcpmon debug counter prints every 18 seconds
                 loganalyzer.analyze(marker)
+                if testing_mode == DUAL_TOR_MODE:
+                    loganalyzer_standby.analyze(marker_standby)
     except LogAnalyzerError as err:
         logger.error("Unable to find expected log in syslog")
         raise err
 
-    # Clean up - Restart DHCP relay service on DUT to recover original dhcpmon setting
-    restart_dhcp_service(duthost)
-    pytest_assert(wait_until(120, 5, 0, check_interface_status, duthost))
+    if not skip_dhcpmon:
+        # Clean up - Restart DHCP relay service on DUT to recover original dhcpmon setting
+        restart_dhcp_service(duthost)
+        if testing_mode == DUAL_TOR_MODE:
+            restart_dhcp_service(standby_duthost)
+            pytest_assert(wait_until(120, 5, 0, check_interface_status, standby_duthost))
+        pytest_assert(wait_until(120, 5, 0, check_interface_status, duthost))
```

Comment:
I don't see dhcp configuration change in branch "if not skip_dhcpmon:", why only recover in this branch?

Reply:
wer're restarting dhcp service here to recover dhcpmon from debug mode from start_dhcp_monitor_debug_counter